public/app/features/teams/CreateTeam.tsx

```diff
@@ -1,51 +1,81 @@
-import React, { PureComponent } from 'react';
+import React, { useState } from 'react';
 
 import { getBackendSrv, locationService } from '@grafana/runtime';
 import { Button, Form, Field, Input, FieldSet } from '@grafana/ui';
 import { Page } from 'app/core/components/Page/Page';
+import { TeamRolePicker } from 'app/core/components/RolePicker/TeamRolePicker';
+import { updateTeamRoles } from 'app/core/components/RolePicker/api';
+import { useRoleOptions } from 'app/core/components/RolePicker/hooks';
 import { contextSrv } from 'app/core/core';
+import { AccessControlAction, Role } from 'app/types';
 
 interface TeamDTO {
-  name: string;
   email: string;
+  name: string;
 }
 
-export class CreateTeam extends PureComponent {
-  create = async (formModel: TeamDTO) => {
-    const result = await getBackendSrv().post('/api/teams', formModel);
-    if (result.teamId) {
-      await contextSrv.fetchUserPermissions();
-      locationService.push(`/org/teams/edit/${result.teamId}`);
+export const CreateTeam = (): JSX.Element => {
+  const currentOrgId = contextSrv.user.orgId;
+  const [pendingRoles, setPendingRoles] = useState<Role[]>([]);
+  const [{ roleOptions }] = useRoleOptions(currentOrgId);
```

Comment:
It looks like the second `setOrgId` return value is never used, should it be removed from the hook or could there be any future usages for it?

Reply:
I would leave `setOrgId` there, because orgId is the only parameter of `fetchRoleOptions` and this hook could be reused somewhere else in the future.